### PR TITLE
pat-385 probation nominal profile

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: server/views/partials/layout.html [hunk 1]
```diff
@@ -20,6 +20,8 @@
 
 {% endblock %}
```
Comment (lbennett-moj) [inline server/views/partials/layout.html]: Not sure what we want the title to be - but its an improvement on the default

File: integration-tests/integration/caseList/view-probation-nominal.spec.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const ProbationNominalPage = require('../../pages/probationNominalHome')
+
+let nominalId
+
+context('Case list and nominal pages', () => {
```
Comment (lbennett-moj) [inline integration-tests/integration/caseList/view-probation-nominal.spec.js]: I'll correct this comment text on the next commit

File: server/views/partials/layout.html [hunk 3]
```diff
@@ -20,6 +20,8 @@
...
 
+{% block pageTitle %}HMPPS Pathfinder{% endblock %}
```
Comment (lbennett-moj) [inline server/views/partials/layout.html]: Not sure what we want the title to be - but its an improvement on the default

File: assets/sass/local.sass [hunk 4]
```diff
@@ -189,6 +189,17 @@ a.no-underline
...
   margin-top: 5px
 
+.solid-border-dark-blue-task-list
+  width: 89.5%
+  height: 28px
+  text-align: center
+  float: left
+  color: white
+  border-style: solid
+  background: 	#00437b
```
Comment (yasinmustafa) [inline assets/sass/local.sass]: bit nit picky - but spacing on line 199 looks odd

File: server/services/offenderService.js [hunk 5]
```diff
@@ -8,30 +8,18 @@ module.exports = function createOffendersService(
...
 ) {
-  const getOffenderFullByBookingId = async (context, bookingId) => {
-    const elite2Client = elite2ClientBuilder(context)
-    const booking = await getBookingData(elite2Client, bookingId)
-    if (isNilOrEmpty(booking)) {
-      logger.warn(`No details found for bookingId: ${bookingId}`)
-      return {}
+  const getOffenderFull = async (context, nominal) => {
+    if (nominal.sourceSystem === SourceSystem.NOMIS) {
+      return getoffenderFullNomis(context, nominal)
```
Comment (yasinmustafa) [inline server/services/offenderService.js]: minor - getoffenderFullNomis not in camel case

File: server/services/offenderService.js [hunk 6]
```diff
@@ -41,13 +29,17 @@ module.exports = function createOffendersService(
...
-    const offenderFull = { ...booking, ...sentence, ...prisoner, ...mainOffence }
-
-    // Keep for now - useful whilst developing this area
-    // const offenderFullStr = JSON.stringify(offenderFull)
-    // logger.info(`OffenderFull: ${offenderFullStr}`)
+    return { ...booking, ...sentence, ...prisoner, ...mainOffence }
+  }
 
-    return offenderFull
+  const getoffenderFullProbation = async nominal => {
```
Comment (yasinmustafa) [inline server/services/offenderService.js]: minor - not in camel case

File: server/services/offenderService.js [hunk 7]
```diff
@@ -8,30 +8,18 @@ module.exports = function createOffendersService(
...
-    // Keep for now - useful whilst developing this area
-    // const offenderFullStr = JSON.stringify(offenderFull)
-    // logger.info(`OffenderFull: ${offenderFullStr}`)
-
-    return offenderFull
+    return getoffenderFullProbation(nominal)
   }
 
-  const getOffenderFullByNomsId = async (context, nomsId) => {
+  const getoffenderFullNomis = async (context, nominal) => {
```
Comment (yasinmustafa) [inline server/services/offenderService.js]: camel case here aswell

**Pull request merged**